.github/workflows/release.yml

```diff
@@ -124,6 +124,7 @@ jobs:
           if [ "$SKIP_PUBLISH" = "false" ]
           then
             echo "Publishing $VERSION to the "$RELEASE_CHANNEL" channel"
+            cp README.md ./projects/optic/README.md
```

Comment:
@notnmeyer not sure how to test this without actually publishing but my "Looks right" passed 1/1

Reply:
yeah, thats the rub on this stuff — hard to test